style.css

```diff
@@ -767,6 +764,10 @@ nav > i:hover {
 /* ----DESKTOP VERSION---- */
 
 @media only screen and (min-width: 768px) {
+  .body {
```

Comment:
- [ ] As mentioned by the previous reviewer, please make sure you fix this sliding effect, your current design is moving from left to right and that makes your app behave badly, please fix it

![5](https://user-images.githubusercontent.com/77843050/213889519-486fcbd0-449b-40e5-b5f9-7dee98e72289.gif)

- [ ] Your Desktop code should not break down the former mobile version design, make sure both your desktop and mobile version work fine, and see how it behave currently

![5](https://user-images.githubusercontent.com/77843050/213889637-671fcc40-492b-4e7c-b3d4-1b23448f70bb.gif)